root/Makefile

```diff
@@ -147,10 +149,16 @@ gobuild: check-deps
 grpcui:
 	@$(LOG) info "Launching gRPCUI"
 	./scripts/shell-wrapper.sh grpcui.sh localhost:5000
+
+## run:           run the service - the right binary, with opt-in logfmt
+.PHONY: run
+run:: pre-run build
+	@DEVBOX_LOGFMT="$(LOGFMT)" ./scripts/shell-wrapper.sh air-runner.sh
+
 ## dev:       run the service
 .PHONY: dev
 dev:: pre-dev
-	@./scripts/shell-wrapper.sh gobin.sh github.com/cosmtrek/air@f7ec5b5ac29540748772dd4deb24a39f81ba9a47 -c $(CURDIR)/.air.toml
+	@DEVBOX_LOGFMT="$(LOGFMT)" ./scripts/shell-wrapper.sh gobin.sh github.com/cosmtrek/air@f7ec5b5ac29540748772dd4deb24a39f81ba9a47 -c $(CURDIR)/.air.toml
 
 ## devserver:       run the service
 .PHONY: devserver
```

Comment:
Should we just alias `devserver` to `run` right now?

Reply:
They are different - in terms of running devconfig stuff.